CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,54 @@
+# Code of Conduct
+
+All developers should feel welcome and encouraged to contribute to Hummingbird. Because of this we have adopted the code of conduct defined by [contributor-covenant.org](https://www.contributor-covenant.org). This document is used across many open source 
```

Comment:
```suggestion
All developers should feel welcome and encouraged to contribute to valkey-swift. Because of this we have adopted the code of conduct defined by [contributor-covenant.org](https://www.contributor-covenant.org). This document is used across many open source 
```